one_fm/overrides/sales_invoice.py

```diff
@@ -76,7 +76,7 @@ def populate_contract_item_categorywise_summary(self):
                 "contract_item_category": row.contract_item_category,
                 "base_net_amount": row.base_net_amount
             }
-            for row in self.get("custom_contract_item_categorywise_summary", [])
+            for row in self.get("custom_contract_item_categorywise_summary", []) or []
```

Comment:
this seems redundant, ad the other argument in the get methof would also return a list